src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -79,7 +79,7 @@ public void completed(AbstractBuild r) {
         Result result = r.getResult();
         AbstractBuild<?, ?> previousBuild = project.getLastBuild();
         do {
-            previousBuild = previousBuild.getPreviousBuild();
+            previousBuild = previousBuild.getPreviousCompletedBuild();
```

Comment:
For reference, [documentation for `getPreviousCompletedBuild()`](http://javadoc.jenkins-ci.org/hudson/model/Run.html#getPreviousCompletedBuild%28%29).  Also, also at first I thought `getPreviousCompletedBuild` meant the last successful build but in general it means the last completed build regardless of its resulting status.